src/Device.ts

```diff
@@ -103,8 +103,10 @@ export function detectDevice(): BuiltinHandlerName | undefined
 		const os = uaParser.getOS();
 		const osName = os.name?.toLowerCase() ?? '';
 		const osVersion = parseFloat(os.version ?? '0');
+		const device = uaParser.getDevice();
+		const deviceModel = device.model?.toLowerCase() ?? '';
 
-		const isIOS = osName === 'ios';
+		const isIOS = osName === 'ios' || (osName === 'mac os' && deviceModel === 'ipad');
```

Comment:
What about just this?

```suggestion
		const isIOS = osName === 'ios' || deviceModel === 'ipad';
```